.pylintrc

```diff
@@ -2,8 +2,7 @@
 ignore-patterns=__main__.py,.*_pb2.py,tac.sh,tac_local.sh
 
 [MESSAGES CONTROL]
-# Skips after C0209 are temporary
-disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209,R1710,R1729,C0206,W0237,R1735,W1505,R1732,W1514,R0402,R1714,R1734,W0640,W0621,E0401,R1733
+disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209
```

Comment:
CS209 must remain for now (it has to do with migrating string formatting to f-strings). 

I've used [flynt](https://pypi.org/project/flynt/) to do 90% automatically and I fixed the rest manually but I found that  the protocol generator will rewrite some of these changes as it uses ".format()" for the messages. The code needs several changes to fix this behaviour, and I've already tried some tests locally but I don't want to expand this PR's scope that should be only about linting. If we want to remove CS209 from here I'll open an issue for that.